XpressAI/xircuits
# Add mouse clicking and hover components

Welcome to Xircuits! Thank you for making a pull request. Please ensure that your pull request follows the template.

# Description

Release v0.2.0: Added clicking and hovering RPA components with demo example.

## References

NA

## Pull Request Type


## Type of Change


# Tests

1. Follow the installation instructions [here](https://xircuits.io/docs/Installation), if Xircuits is not installed yet.
2. Launch Xircuits.
    `$ xircuits`
3. Navigate to Xircuits Component Library at the sidebar and make sure RPA folder is available and contains necessary components - `RpaClick` and `RpaHover`.
4. Navigate to File Browser > examples and open `RpaClickAndHover.xircuits`.
5. Compile the `.xircuits` file and execute it.
6. You should see something like the following:
```
Xircuits is running...


Executing: RpaInit
Visual automation: True, Chrome: True, Turbo: False
Bot initiated.

Executing: RpaUrl
Opening https://www.vecto.ai/ on browser...
Browser opened with URL https://www.vecto.ai/.

Executing: RpaClick
Clicking element See it in Action...
Clicked element See it in Action.

Executing: RpaClick
Clicking element Find nearest neighbors...
Clicked element Find nearest neighbors.

Executing: RpaClick
Clicking element Looking up by analogy...
Clicked element Looking up by analogy.

Executing: RpaHover
Hovering on element s_social_media_github...
Hovered on element s_social_media_github.

Executing: SleepComponent
Sleeping for 5.0 seconds.

Executing: RpaClose
Closing RPA...

Finished Executing
```

## Tested on?


# Notes

NA
